src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -104,6 +125,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapControllers();
+                endpoints.MapHealthChecks("/health/live", new HealthCheckOptions
+                {
+                    Predicate = (healthCheckRegistration) => healthCheckRegistration.Tags.Contains(HealthCheckTags.Live)
```

Comment:
The brackets can be omitted from the lambda parameter here.

Reply:
👍